### ISSUE TITLE
implied versus observed table?

### ISSUE THREAD

**Issue by jrosen48** [jrosen48/konfound#26]
For our non-linear output, we use the term "implied table"
```
For the Implied Table, we have an estimate of -0.200, with a standard error of 0.103 and a t-ratio of -1.946.

For the Transfer Table, we have an estimate of -0.202, with a standard error of 0.103 and a t-ratio of -1.963.
```

For our `tkonfound()` output, we use the term "observed table"

```
For the Observed Table (based on user enter values), we have a Pearson's chi square of 2.046, with p value of 0.153.

For the Transfer Table, we have a Pearson's chi square of 3.851, with p value of 0.050.
```

just confirming this is right!

**Comment by linqinyu** [comment]
Yes! Because the tkonfound uses the user-entered value for the 2x2 table. In the nonlinear output, we calculated an implied 2x2 table based on the estimated effect and standard error (and other information) entered by the user. In other words, the table is not directly given by the user, but rather we calculated based on other information given by the user. In fact, based on the estimated effect and other information, we can have more than two possible solutions for the 2X2 table. Does this make sense? 
But you reminded me of a possibility that the user gave a 2x2 table in the tkonfound that is not observed.  In that case, it is more precise just to say the table based on user-entered value. em...I think I will remove "Observed" to be clear! Thanks!!
